tests/e2e/testcases/upgrade/suite.go

```diff
@@ -185,9 +245,8 @@ var _ = ginkgo.Describe("[Upgrade]", ginkgo.Ordered, func() {
 	})
 
 	ginkgo.It("upgrade SubnetEVM local deployment", func() {
-		// create and deploy
 		commands.CreateSubnetEvmConfigWithVersion(subnetName, utils.SubnetEvmGenesisPath, subnetEVMVersion1)
-		deployOutput := commands.DeploySubnetLocally(subnetName)
+		deployOutput := commands.DeploySubnetLocallyWithVersion(subnetName, avagoRPC1Version)
```

Comment:
This change is not needed test will work anyway

Reply:
Yes you provided proof that it does. I suggest that we still keep that, because in these tests all too many things are happening below the surface and they are difficult to debug. This makes it explicit with what version we are deploying.